Arashayo/TabletopHaven
# Add other user profiles

## Related Issue
https://app.clickup.com/t/86bxevw15

## Changes list
Added other people profiles
Added following functionality

## Testing instructions

1. Run both backend and frontend
2. Log in with 1 account
3. Log out
4. Log in with another account
5. Go to http://localhost:8081/user/1 and http://localhost:8081/user/2    these should be both of your accounts
6. Try following and unfollowing each other
7. Check if you can follow yourself (you shouldn't be able to)
8. Check if you can access the profiles when you are not logged in (you should be able to, but you shouldn't have the option to follow)

## Screenshots (if appropriate):
